recipes/libiconv/all/conanfile.py

```diff
@@ -131,6 +132,8 @@ def generate(self):
             env.define("RANLIB", ":")
             env.define("NM", "dumpbin -symbols")
             env.define("win32_target", "_WIN32_WINNT_VISTA")
+        elif self.settings.os == "Android":
+            env.define("LD", "ld")
```

Comment:
@franramirez688 must `required_conan_version` be raised to 2.11 because of this change?